src/Controller/PermissionController.php

```diff
@@ -34,18 +34,33 @@ public function addAction(Application $app, Request $request, EventDispatcherInt
         $space = $app->getSpaceRepository()->findByNameAndAccountName($spaceName, $accountName);
 
         $error = null;
-        if ($space) {
-            $repo = $app->getPermissionRepository();
-            $error = $repo->add($username, $space->getId(), $roles);
-            $admin = $app['current_user']->getName();
-            $event = new PermissionDomain\PermissionGrantedEvent(
+
+        $client = $app['userbase.client'];
+        try {
+            $user = $client->getUserByUsername($username);
```

Comment:
@kaushikindianic `$username` is the posted value. we should check `$app['current_user']`. 
For now, as long as the user is logged in, we can allow the index and add actions.